### PR TITLE
[no ticket][risk=no] Puppeteer: Use PascalCase for const and enum names

### PR DESCRIPTION
**Pull request by aweng98**
typescript constant name is in PascalCase. See https://www.typescriptlang.org/docs/handbook/enums.html#enums

For consistency and standard, rename Const and Enum names in e2e project to PascalCase (aka. Upper CamelCase).
There are lot of names in `workspace-*-page.ts` which I did not rename in this PR. There will be another PR to cover rest of names for consistency in future.

### CODE REVIEW

File: e2e/app/element/xpath-defaults.ts [hunk 1]
```diff
@@ -56,6 +56,9 @@ export function xPathOptionToXpath(xOpts: XPathOptions, container?: Container):
   case ElementType.Dropdown:
     selector = `${textExpr}//*[contains(concat(" ", normalize-space(@class))," p-dropdown")]`;
     break;
+  case ElementType.Tab:
+    selector = `//*[(@aria-selected | @tabindex) and @role="button" and text()="${name}"]`;
+    break;
```
Comment (aweng98) [inline e2e/app/element/xpath-defaults.ts:61]: Unrelated to rename. But it is a small change.

File: e2e/app/page/workspaces-page.ts [hunk 2]
```diff
@@ -66,7 +64,7 @@ export default class WorkspacesPage extends WorkspaceEditPage {
   * 4: return
   */
   async clickCreateNewWorkspace(): Promise<WorkspaceEditPage> {
-    const link = await Button.findByName(this.page, FIELD.createNewWorkspaceButton.textOption);
+    const link = await Button.findByName(this.page, FieldSelector.CreateNewWorkspaceButton.textOption);
```
Comment (NehaBroad) [inline e2e/app/page/workspaces-page.ts:67]: nit: indent

File: e2e/app/page/google-login.ts [hunk 3]
```diff
@@ -5,13 +5,13 @@ import {waitForDocumentTitle} from 'utils/waits-utils';
...
 
-export const SELECTOR = {
-  loginButton: '//*[@role="button"]/*[contains(normalize-space(text()),"Sign In")]',
-  emailInput: '//input[@type="email"]',
-  NextButton: '//*[text()="Next" or @value="Next"]',
-  submitButton: '//*[@id="passwordNext" or @id="submit"]',
-  passwordInput: '//input[@type="password"]',
-};
+export enum FieldSelector {
+  LoginButton= '//*[@role="button"]/*[contains(normalize-space(text()),"Sign In")]',
```
Comment (NehaBroad) [inline e2e/app/page/google-login.ts:9]: nit: space after loginbutton
